src/Event/FieldEncryptingConverter.php

```diff
@@ -0,0 +1,77 @@
+<?php
+declare(strict_types=1);
+
+namespace MyOnlineStore\EventSourcing\Event;
+
+use Zend\Crypt\BlockCipher;
```

Comment:
This dependency would benefit from an abstraction: the specific encryption library need not be locked into this base solution